routes/web.php

```diff
@@ -20,3 +20,7 @@
 
 Route::get('/home', 'HomeController@index')->name('home');
 
+Route::resource('courses', 'CourseController')->names([
+   'index' => 'bootcamp',
```

Comment:
same again, I would have just called this `courses` as it is the index for courses.